databuilder/models/badge.py

```diff
@@ -65,6 +60,11 @@ def __init__(self,
         self._node_iter = iter(self.create_nodes())
         self._relation_iter = iter(self.create_relation())
 
+    def __repr__(self) -> str:
+        return 'BadgeMetadata({!r}, {!r}, {!r})'.format(self.cluster,
```

Comment:
is cluster being used anywhere?

Reply:
no it is not but I did not remove it thinking if we want different badge for different clusters. I can remove it though since w e are not storing this info anywhere

Reply:
I don't think we need it given the start_key uri should include cluster field (`db://cluster.schema/table`?) even if we have different clusters, the uri will be different?